packages/ns-api/files/ns.ovpnrw

```diff
@@ -882,6 +886,120 @@ def download_user_2fa(ovpninstance, username):
     else:
         return utils.validation_error("username", "2fa_download_failed", username)
 
+def _get_database(ovpninstance):
+    # We try the ram path, in the future we could have a storage path, if fail we return an empty string
+    db_path = [ f'/var/openvpn/{ovpninstance}/connections.db']
+    database_path = ""
+    for db in db_path:
+        if os.path.isfile(db):
+            database_path = db
+            break
+    return database_path
+
+def connection_history_csv(ovpninstance):
+    database_path = _get_database(ovpninstance)
+    if database_path == "":
+        return {}
+
+    conn = sqlite3.connect(database_path)
+    c = conn.cursor()
```

Comment:
Add a try/except here, in case of error return a https://nethserver.github.io/python3-nethsec/nethsec.utils.html#generic_error

Reply:
done